skycultures/arabic/star_names.fab

```diff
@@ -0,0 +1,179 @@
+ 67459|_("Flag Of The High One")
+# TRANSLATORS: Al-Fakkah is an incomlete circle
+ 76267|_("The Bright Star Of Al-Fakkah")
+ 91262|_("The Landing Eagle")
+ 89826|_("The Landing Eagle Claws 1")
+ 90191|_("The Landing Eagle Claws 2")
+102098|_("The Rear Knight")
+ 94779|_("The Knights 1")
+ 97165|_("The Knights 2")
+100453|_("The Knights 3")
+102488|_("The Knights 4")
+104732|_("The Knights 5")
+ 18246|_("The Shoulder Blade")
+ 18614|_("The Shoulder")
+ 18532|_("The Arm")
+ 15863|_("The Elbow")
+ 16335|_("Bend Of The Hand")
+ 17358|_("Tip Of The Elbow")
+ 14328|_("The Forearm")
+ 11020|_("The Wrist")
+# TRANSLATORS: Al-Ayyuq is a proper name meaning the obstructor
+ 24608|_("Al-Ayyuq")
+# TRANSLATORS: Ayyuq Al-Thurayya is a proper name meaning the obstructorof Al-Thurayya
+ 24608|_("Ayyuq Al-Thurayya")
+# TRANSLATORS: Al-Thurayya is a proper name
+ 24608|_("Al-Thurayya watcher")
+ # TRANSLATORS: Al-Ayyuq is a proper name meaning the obstructor
+ 23015|_("Foot Of Al-Ayyuq")
+ 28360|_("The Posts 1")
+ 28380|_("The Posts 2")
+ 25428|_("The Posts 3")
+ 28360|_("The Associates Of Al-Ayyuq 1")
+ 28380|_("The Associates Of Al-Ayyuq 2")
+ 25428|_("The Associates Of Al-Ayyuq 3")
+ 86032|_("The [Southern] Shepherd")
+ 84345|_("First Dog Of The [Southern] Shepherd")
+ 86742|_("Second Dog Of The [Southern] Shepherd")
+ 97649|_("The Flying Eagle")
+101421|_("The Post Of The Cross") 
+   677|_("The Rear Bucket Mouth 2")
+  1067|_("The Rear Bucket Mouth 1")
+113881|_("The Front Bucket Mouth 2")      
+113963|_("The Front Bucket Mouth 1")      
+   677|_("The Lower Cross Beam 2")
+  1067|_("The Lower Cross Beam 1")
+113881|_("The Upper Cross Beam 2")
+113963|_("The Upper Cross Beam 1")
+  5447|_("Abdomen Of The Fish")
+  9640|_("The Caracal")
+  8796|_("The Two Forelegs 1")
+ 10064|_("The Two Forelegs 2")
+ 21421|_("Al-Dabaran")
+ 21421|_("The Male Camel")
+ 21421|_("The Wood Stirring Rod")
+ 21421|_("The Camel Herder")
+ 29655|_("Stars Of Abundant Rain 1")
+ 30343|_("Stars Of Abundant Rain 2")
+ 30883|_("Stars Of Abundant Rain 3")
+# TRANSLATORS: Az-zir is a proper name
+ 31681|_("Az-Zir")
+# TRANSLATORS: Al-Maysan is a proper name
+ 32362|_("Al-Maysan")
+ 36046|_("The Claws 1")
+ 36962|_("The Claws 2")
+ 37740|_("The Claws 3")
+ 35550|_("The Claws 4")
+ 34088|_("The Claws 5")
+ 35350|_("The Claws 6")
+ 46750|_("The Eyelashes")
+ 48455|_("The Eyelashes")
+ 46771|_("The Eyelashes")
+ 46774|_("The Eyelashes")
+ 57632|_("Star Of Weather Change")
+ 65474|_("The Unarmed High One")
+ 80763|_("The Heart")
+ 80112|_("Veins Of The Heart 1")
+ 81266|_("Veins Of The Heart 2")
+ 82396|_("Tail Segments Of The Scorpion 1")
+ 82514|_("Tail Segments Of The Scorpion 2")
+ 82729|_("Tail Segments Of The Scorpion 3")
+ 84143|_("Tail Segments Of The Scorpion 4")
+ 86228|_("Tail Segments Of The Scorpion 5")
+ 87073|_("Tail Segments Of The Scorpion 6")
+ 86670|_("Tail Segments Of The Scorpion 7")
+ 85927|_("Raised Tail Of The Scorpion")
+# 85696|_(" ")

```

Comment:
Why it empty?

Reply:
It is a comment.
A common name is given to HIP 85927 and HIP 85696. The two stars are very close to each other, so one label is used to mimimize clutter and I kept the other as a comment for documentation.